x-pack/plugins/spaces/server/routes/api/external/copy_to_space.ts

```diff
@@ -77,19 +84,23 @@ export function initCopyToSpacesApi(deps: ExternalRouteDeps) {
     },
     createLicensedRouteHandler(async (context, request, response) => {
       const [startServices] = await getStartServices();
-
-      const copySavedObjectsToSpaces = copySavedObjectsToSpacesFactory(
-        startServices.savedObjects,
-        getImportExportObjectLimit,
-        request
-      );
       const {
         spaces: destinationSpaceIds,
         objects,
         includeReferences,
         overwrite,
         createNewCopies,
       } = request.body;
+
+      const { headers } = request;
+      const usageStatsClient = await usageStatsClientPromise;
+      await usageStatsClient.incrementCopySavedObjects({ headers, createNewCopies, overwrite });
```

Comment:
there's no need to block this API call until usage stats are persisted, we should rather not await this promise (especially with the default `refresh=wait_for` this can make routes a lot slower (see other comment)